frontend/packages/jobs/src/functions/processGenerateDocsSuggestion.ts

```diff
@@ -52,7 +52,15 @@ export async function processGenerateDocsSuggestion(payload: {
       return {
         id: filename,
         title: filename,
-        content: fileData.content || '',
+        content: fileData.content
+          ? Buffer.from(fileData.content, 'base64')
+              .toString('utf-8')
+              .replace(/\\/g, '\\\\')
+              .replace(/\n/g, '\\n')
+              .replace(/\r/g, '\\r')
+              .replace(/\t/g, '\\t')
+              .replace(/"/g, '\\"')
+          : '',
```

Comment:
As pr-agent https://github.com/liam-hq/liam/pull/1278#issuecomment-2789548687 said, this is like `JSON.stringify` . To handle other special characters, you may want to consider using `JSON.stringify` and `split(1,-1)`.

Reply:
Thank you very much! You were right. It is simpler now.🙏（Confirmed that there is no problem locally.）

https://github.com/liam-hq/liam/pull/1278/commits/deafc888c673a112c57473711f611e47e99378e2